src/main/java/org/jabref/logic/pdf/PdfAnnotationImporter.java

```diff
@@ -71,6 +67,10 @@
     }
 
     private boolean isSupportedAnnotationType(PDAnnotation annotation) {
+        if (annotation.getSubtype().equals("Link") || annotation.getSubtype().equals("Widget")) {
```

Comment:
Good point, should not be the case, but that pdfbox library is really not to be trusted....